modules/nav.js

```diff
@@ -0,0 +1,28 @@
+import {
+  booksSec, bookCreate, booksLink, contact, addBookLink, contactLink,
+} from './variables.js';
+
+const navigation = () => {
+  booksLink.addEventListener('click', (e) => {
+    e.preventDefault();
+    booksSec.style.display = 'block';
+    bookCreate.style.display = 'none';
+    contact.style.display = 'none';
+  });
+
+  addBookLink.addEventListener('click', (e) => {
+    e.preventDefault();
+    bookCreate.style.display = 'block';
+    booksSec.style.display = 'none';
+    contact.style.display = 'none';
+  });
+
+  contactLink.addEventListener('click', (e) => {
+    e.preventDefault();
+    bookCreate.style.display = 'none';
+    booksSec.style.display = 'none';
+    contact.style.display = 'block';
+  });
```

Comment:
- [ ] Great job on working on the requirements so far but I think you had an error on the functionality while refactoring your code, so please solve the errors to let your project works as expected ⭐💯😊✨🤩

![image](https://user-images.githubusercontent.com/58553711/220233758-1869e517-a312-49a0-bb05-5d6e550d2076.png)